src/docker/Dockerfile

```diff
@@ -56,39 +57,16 @@ RUN apt-get install -y software-properties-common python-software-properties
 RUN add-apt-repository ppa:git-core/ppa
 RUN apt-get update && apt-get install -y git git-core
 
-# SSH Daemon
-RUN apt-get install -y ssh && \
-  mkdir /var/run/sshd && \
-  chmod 0755 /var/run/sshd
-
-# Ruby
-RUN apt-get install -y libmagic-dev zlib1g-dev
-RUN apt-add-repository ppa:brightbox/ruby-ng
-RUN apt-get update
-RUN apt-get install -y ruby2.2 ruby2.2-dev
```

Comment:
@amihaiemil I don't think you can drop the ruby section, it's needed for the puzzle creation. You can make this a little nicer though (see below comment why this is important), by joining lines 67 and 68 into `RUN apt-get update && apt-get install -y ruby2.2 ruby2.2-dev`